frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -492,24 +492,26 @@ describe("scenarios > question > custom columns", () => {
     cy.button("Done").should("not.be.disabled");
   });
 
-  describe.skip("contentedtable field (metabase#15734)", () => {
+  describe("contentedtable field (metabase#15734)", () => {
     beforeEach(() => {
       // This is the default screen size but we need it explicitly set for this test because of the resize later on
       cy.viewport(1280, 800);
 
       openOrdersTable({ mode: "notebook" });
       cy.findByText("Custom column").click();
-      popover().within(() => {
-        cy.get("[contenteditable='true']")
+
+      cy.get("[contenteditable='true']")
           .as("formula")
```

Comment:
There's no point in aliasing `contenteditable` if we're not going to use it later in the test.
Btw, the reason I had to use `popover().within(...)` was because of some race condition (my best guess). It proved to be the pattern that somewhat helps with the problem. Whenever I tried to go directly for the contenteditable and start typing, I've seen flakes and weird behavior.

Reply:
> There's no point in aliasing contenteditable if we're not going to use it later in the test.

We use `@formula` later in 3 tests inside this `describe`.

Reply:
I am sorry.
Totally overlooked the fact that this is inside a `beforeEeach()` hook. My bad.